packages/frontend/src/components/ActionPanel/ActionPanel.tsx

```diff
@@ -37,36 +49,36 @@ interface ActionPanelProps {
 export function ActionPanel({ actionButtons, onUndoSelection, selectedItemCount = 0 }: ActionPanelProps) {
   const { t } = useTranslation();
   return (
-    <div className={styles.actionPanel}>
-      <div className={styles.actionButtons}>
-        {actionButtons
-          .filter((actionBtn) => actionBtn.hidden !== true)
-          .map(({ label, onClick, icon, disabled }) => {
-            return (
-              <Button
-                className={styles.actionButton}
-                key={label}
-                onClick={onClick}
-                disabled={disabled}
-                variant="tertiary"
-                size="small"
-              >
-                <span className={styles.actionButtonIcon}>{icon}</span>
-                <span className={styles.actionButtonLabel}>{label}</span>
-              </Button>
-            );
-          })}
-      </div>
-      {selectedItemCount > 0 && (
-        <div>
-          <Button className={styles.undoButton} onClick={onUndoSelection} variant="tertiary" size="small">
-            <span className={styles.undoButtonLabel}>{t('actionPanel.chosen', { count: selectedItemCount })}</span>
-            <span className={styles.undoButtonIcon}>
-              <XMarkIcon />
-            </span>
-          </Button>
+    <>
```

Comment:
</> ikke nødvendig, vel? 🗡️

Reply:
Retunerer to ting, en BulkHeader og en BulkFooter. Så må ha noe som wrapper dem.

Reply:
Han har to komponenter så de må wrappes inni noe, og bedre å bruke en Fragment (som han gjør) enn en <div> imo